### PR TITLE
Shortlisted Dashboard update

### PR DESCRIPTION
**Pull request by dreamfall**
https://app.asana.com/0/1200504523179345/1202154074583687

- add new mailers for mobility/PO forms
- add new file upload for mobility/po forms
- change the dashboard layout
- add ability to review new shortlisted doc type for admin/assessor

### CODE REVIEW

File: app/models/form_answer.rb [hunk 1]
```diff
@@ -161,6 +164,9 @@ def secondary
     scope :primary_and_secondary_appraisals_are_not_match, -> {
       where("discrepancies_between_primary_and_secondary_appraisals::text <> '{}'::text")
     }
+
+    scope :require_vocf, -> { where(award_type: %w[trade innovation]) }
```
Comment (TheDancingClown) [inline app/models/form_answer.rb:168]: Do you also need to check the award year (pre 2023) and if busines in this scope?

File: app/models/shortlisted_documents_wrapper.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  private
+
+  def valid_for_submission?
+    return true unless submitted?
+
+    if vat_returns_files.first&.attachment.present?
+      true
+    else
+      errors.add(:base, "You need to add at least one VAT returns file")
```
Comment (TheDancingClown) [inline app/models/shortlisted_documents_wrapper.rb:35]:
```suggestion
      errors.add(:base, "You need to add at least one VAT return file")
```

**Review by TheDancingClown — APPROVED** [review]

File: app/models/form_answer.rb (hunk 1 above)
Comment (dreamfall) [reply app/models/form_answer.rb:168]: I think it's gonna be fine like that, moving forward it's only gonna be used on applications > 2023.

File: app/models/shortlisted_documents_wrapper.rb (hunk 2 above)
Comment (dreamfall) [reply app/models/shortlisted_documents_wrapper.rb:35]: I think it should be `VAT returns` cause the input is named `Upload VAT returns`

**Pull request merged**